recommendations.json

```
@@ -9,6 +9,17 @@
       }
+    },
+    {
+      "extensionId": "redhat.ai-lab",
+      "publishDate": "2024-05-06",
```

```suggestion
      "publishDate": "2024-05-06T16:00:00.000Z",
```

See https://github.com/containers/podman-desktop-internal/issues/243#issuecomment-2077558289